benefits/core/templates/core/base.html

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Thank you for deleting this! It always bothered me, couldn't figure out why it was there :sweat_smile: